_data/apps.yml

```diff
@@ -1848,3 +1848,12 @@
     - "programming"
     - "productivity"
   icon: "cycligent.png"
+-
+  name: "Mixmax"
+  description: "See every action on your emails in real-time. Compose anywhere."
+  website: "https://mixmax.com/download"
+  keywords:
+    - "email"
+    - "analytics"
+  icon: "mixmax.png"
+  featuredOnHomepage: true
```

Comment:
http://electron.atom.io/apps has all of the apps that have been addd (nearly 200 now) the front page just shows 20.